hack/gen-versions/components.go

```diff
@@ -31,6 +31,8 @@ var defaultImages = map[string]string{
 	"calicoctl":                  "calico/ctl",
 	"flannel":                    "coreos/flannel",
 	"flexvol":                    "calico/pod2daemon-flexvol",
+	"calico/csi":                 "calico/csi",
+	"csi-node-driver-registrar":  "csi-node-driver-registrar",
```

Comment:
I'm thinking the value should be `sig-storage/csi-node-driver-registrar` and then remove `sig-storage` from the registry?

Reply:
That makes sense to be more consistent with the others.